app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -14,12 +14,12 @@ const phoneUtil = libphonenumber.PhoneNumberUtil.getInstance();
 class Contact extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { ...this.props, current: { ...this.props.currentState }, errors: {}, modified: {}, isEditMode: window.location.href.includes('edit') };
+    this.state = { ...this.props, current: { ...this.props.currentState }, errors: {}, modified: {}, isedit_mode: window.location.href.includes('edit') };
```

Comment:
I think these changes snuck in from elsewhere.

Reply:
yep, it snuck in when I was trying to fix conflicts for a rebase, fixed here: https://github.com/SaraAlert/SaraAlert/pull/689/commits/79f552e73ee0c551a9718f213a2a3d95ded2e0d3